src/tagmanager/tm_workspace.c

```diff
@@ -477,7 +460,7 @@ static GList *lookup_includes(const gchar **includes, gint includes_count)
 	size_t idx_glob;
 #endif
 
-	table = g_hash_table_new_full(tm_file_inode_hash, g_direct_equal, NULL, g_free);
+	table = g_hash_table_new_full(g_str_hash, g_direct_equal, NULL, g_free);
```

Comment:
Should the 2nd parameter be `g_str_equal`, or is it meant to compare the address/pointer of the keys?

Reply:
It probably does not make much of a difference, so I updated it to `g_str_equal`

Reply:
It does in fact matter, the key is a newly copied string, and you need to compare the strings, not the pointers to the strings so `g_str_equal` is right.